### DISCUSSION TITLE
Aggregating other events

### DISCUSSION THREAD

**Discussion by way0utwest** [dataplat/DataSaturdays#128] [Ideas]
The WIT group has posted an event, Data Platform WIT Day - https://wit.sqlugs.com/2021/02/04/data_platform_wit_day/

My question/discussion here is that I'd like to post this as a Data Saturday,  starting to think about DS as more of an umbrella. I think we could list it as one of these two things:
- Data Saturdays #8 - Data Platform WIT Day
- Data Platform WIT Day 

This would be a similar discussion or debate on how to list something like a SQL Saturday or Power BI Saturday or Database Dev Day, which could be:

- Data Saturdays #10 - Denver
- SQL Saturday #1033 - Salt Lake City
- Data Saturdays #11 - Albuquerque
- Data Saturdays #12 - Dallas
- Power BI Saturdays #1 - Redmond

Two questions:
1. Do we list other events like this that are related to help them get exposure and grow this brand?
2. Do we list other events as Data Saturdays or as their own brand?

Appreciate thoughts here.

**Comment by rabryst** [comment]
1. Yes.
2. Keep the names they’ve chosen, or do a co-brand.

**Reply by SQLClause** [reply]
1. Yes - I think we should list other events here. It's a win-win scenario in my opinion :)
2. Let's keep the names they have chosen and rather do co-branding - that builds up the umbrella approach of the Data Saturdays nicely.

**Comment by ceedubvee** [comment]
1. Yes - it helps show the extent of the network for data professionals and worthwhile branding.
2. I'd prefer keeping the names, but a co-brand umbrella could work as pointed out above. For co-branding, would the format be something like this: Power BI Saturday (A Data Saturday event), or potentially like this: Data Saturday, WIT Edition? Wondering the best way to integrate the two while keeping their names prominent.

**Reply by SQLClause** [reply]
I think a  co-branding approach like this suggestion from Steve works: Data Saturdays #8 - Data Platform WIT Day
The event can keep both or just their own on their branding.
